contracts/CourtRoom.sol

```diff
@@ -146,16 +146,17 @@ contract SwearGame is Owned {
     if (decision == true){
 	    caseCompensated = compensate(plaintiff);
 			caseContract.resolveClaim(_id);
+      leaveGame(plaintiff);
```

Comment:
I am still unsure of why a compensated case results in the plaintiff leaving the game. Surely this is not always going to be the desired outcome?